app/services/classroom_config.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+class ClassroomConfig
+  CONFIGURABLES   = %w[issues]
```

Comment:
Hrm... I'm almost positive that we don't need to freeze strings anymore. @tarebyte do you know?